<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogOpenConifg</class>
 <widget class="QDialog" name="DialogOpenConifg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>862</width>
    <height>670</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Open or create a configuration</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="buttonBrowseAndOpen">
       <property name="toolTip">
        <string>Open a configuration by browsing directories. You can do this to open a configuration not in the list below.</string>
       </property>
       <property name="text">
        <string>Browse and open</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelConfigInfo">
       <property name="text">
        <string>Select a new empty directory in which your work will be saved</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_4">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="labelRecentlyOpened">
       <property name="text">
        <string>Recently opened</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonOpenRecent">
       <property name="toolTip">
        <string>Open a configuration recently opened from the list below</string>
       </property>
       <property name="text">
        <string>Open selected</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonClearRecentSelected">
       <property name="toolTip">
        <string>Clear the selected directory from the list below</string>
       </property>
       <property name="text">
        <string>Clear selected</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="Line" name="line">
       <property name="orientation">
        <enum>Qt::Orientation::Vertical</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonReadGtin">
       <property name="toolTip">
        <string>Read GTIN codes from GS1 results report</string>
       </property>
       <property name="text">
        <string>Read GTIN</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QListWidget" name="listRecentPaths">
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectionBehavior::SelectRows</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <item>
      <spacer name="horizontalSpacer_5">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="buttonClose">
       <property name="text">
        <string>Close</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonClose</sender>
   <signal>clicked()</signal>
   <receiver>DialogOpenConifg</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>804</x>
     <y>630</y>
    </hint>
    <hint type="destinationlabel">
     <x>836</x>
     <y>700</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
